ExomeDepth/settings.py.slangenspar

```diff
@@ -66,3 +68,10 @@ igv_settings = {"ratio":[0, 1, 2], "log2ratio":[-2.5, 0, 2.5] }
 correlation = 0.98
 number_calls = [35,200]
 del_dup_ratio = [15,85] 
+
+reanalysis_dic = {
+    "XXY":["female", "XXYMaleAsFemale"],
+    "X0": ["male", "X0FemaleAsMale"],
+    "asFemale": ["female", "AsFemale"],
+    "asMale": ["male", "AsMale"]
+    }
```

Comment:
Different settings on slangespar? `X0`?

Reply:
Should have been removed, thanks. 0 is not a dummy proof character ;)